### ISSUE TITLE
Order of Tests & ReadMe

### ISSUE THREAD

**Issue by itzsaga** [learn-co-curriculum/react-props-and-state-lab#14]
While this might just be a personal preference it might be something worth changing. Since:
> Props are given to the component by its parent. You can think of props as an external influence that the component has no control over

I think it's easier to logically define the props first, then move on to the component that will utilize those props. I found working back towards the source of the props in `App.js` hard to follow. As I was coding the `App.js` it all came together and I could follow the flow of data much better. It might help us as we learn to structure this lab to "follow the data" as it flows from component to component.

**Comment by bhabig** [comment]
I agree with you Seth

**Comment by itzsaga** [comment]
Coming back to this, working with React daily now I think this is even truer. The unidirectional data flow that React provides is a core concept. Being taught to follow that data flow pattern would be best in my humble opinion.

**Comment by realAndrewCohn** [comment]
addressed by #19
